ipfx/x_to_nwb/hr_segments.py

```diff
@@ -324,7 +301,7 @@ def __str__(self):
                  ).format(self.amplitude, self.startFreq, self.endFreq, self.kind)
 
     def getAmplitude(self, channelRec, segmentRec):
-        return channelRec.Chirp_Amplitude
+        return channelRec.Chirp_Amplitude * 0.5
```

Comment:
can you add a comment explaining this constant?

Reply:
@dyf Done. I've mentioned it in the commit message as well btw.